src/utils/utilities.jl

```diff
@@ -94,6 +94,28 @@ function compute_strain_rate!(ϵ::Fields.Field, u::Fields.Field)
         ) / 2
 end
 
+"""
+    compute_strain_rate_face!(ϵ::Field, u::Field)
+
+Compute the strain_rate at cell faces, storing in `ϵ` from
+velocity at cell centers.
+"""
+function compute_strain_rate_face!(ϵ::Fields.Field, u::Fields.Field)
+    @assert eltype(u) <: C123
+    ∇ᵥuvw_boundary =
+        Geometry.outer(Geometry.WVector(0), Geometry.UVWVector(0, 0, 0))
```

Comment:
Will it always be zero BC?

Reply:
I think so. What we have here needs to be consistent with surface flux, which is applied elsewhere, so it should be zero here. If Simon combines the two operations he mentioned above, then whatever we have here will be ignored, as the boundary condition of the `ᶜdivᵥ` operator will override this.